### PR TITLE
Docs: CoC violation correspondants

### LINKED ISSUE #6428: code of conduct enforcement contacts are not publicly visible
in the readme:

>  If you encounter someone violating these terms, please let a [team captain](https://github.com/orgs/jekyll/teams/affinity-team-captains/members) know and we will address it as soon as possible.

that link redirects to the github team page that lists the affinity team captains. however team pages, even when marked as visible, are not visible to anyone _outside_ of the github organization. true, it's fairly easy to join it due to our teams program, but it's still a hurdle i'd rather not see.

also, why does it state the affinity team captains? members of that team itself don't have any special github rights, so all they'd be able to do in terms of enforcement would be to pass it on to core team members. shouldn't there be a link or listing of the core team members instead?

### PR DESCRIPTION
**Pull request by DirtyF**
fix #6428

@pup you seem to be the right person to handle this, maybe we should add another team mate to second you?

### CODE REVIEW

**Comment by pup** [comment]
yeah, definitely don't single me out. at least two people would be perfect. does anyone want to volunteer?

File: README.markdown [hunk 1]
```diff
@@ -45,7 +45,7 @@ conduct.
 Please adhere to this code of conduct in any interactions you have in the
 Jekyll community. It is strictly enforced on all official Jekyll
 repositories, websites, and resources. If you encounter someone violating
-these terms, please let a [team captain](https://github.com/orgs/jekyll/teams/affinity-team-captains/members) know and we will address it as soon as possible.
+these terms, please let our [correspondant](https://github.com/pup) know and we will address it as soon as possible.
```
Comment (pup) [inline README.markdown]: oh, preferably we could put my email address in here (`olivia@fastmail.com`)

**Comment by pathawks** [comment]
Feel free to include me as a contact for CoC violations.

File: README.markdown [hunk 2]
```diff
@@ -45,7 +45,7 @@ conduct.
 Please adhere to this code of conduct in any interactions you have in the
 Jekyll community. It is strictly enforced on all official Jekyll
 repositories, websites, and resources. If you encounter someone violating
-these terms, please let our [correspondant](https://github.com/pup) know and we will address it as soon as possible.
+these terms, please let one of our core team members [Oliva](mailto:olivia@fastmail.com?subject=Jekyll%20CoC%20Violation) or Pat(mailto:pat@pathawks.com?subject=Jekyll%20CoC%20Violation) know and we will address it as soon as possible.
```
Comment (pup) [inline README.markdown]: `Oliva` -> `Olivia` 😄

Comment (DirtyF) [reply README.markdown]: Oops 😕

**Comment by DirtyF** [comment]
I think we're good with three contacts for any CoC violation. 
Jekyll has now its own League of Justice ✊

File: README.markdown [hunk 3]
```diff
@@ -45,7 +45,7 @@ conduct.
 Please adhere to this code of conduct in any interactions you have in the
 Jekyll community. It is strictly enforced on all official Jekyll
 repositories, websites, and resources. If you encounter someone violating
-these terms, please let a [team captain](https://github.com/orgs/jekyll/teams/affinity-team-captains/members) know and we will address it as soon as possible.
+these terms, please let one of our core team members [Olivia](mailto:olivia@jekyllrb.com?subject=Jekyll%20CoC%20Violation), [Pat](mailto:pat@jekyllrb.com?subject=Jekyll%20CoC%20Violation), [Matt](mailto:matt@jekyllrb.com?subject=Jekyll%20CoC%20Violation) or [Parker](mailto:parker@jekyllrb.com?subject=Jekyll%20CoC%20Violation) know and we will address it as soon as possible.
```
Comment (mattr-) [inline README.markdown]: I'm not sure that `jekyllrb.com` email addresses are set up or are they forwarded to our other emails?

Comment (DirtyF) [reply README.markdown]: @mattr-  @parkr kindly offered to configure this. 📫

Comment (mattr-) [reply README.markdown]: oh, excellent. i missed that comment in the other issue.

**Comment by mattr-** [comment]
@jekyllbot: merge +dev

File: README.markdown (hunk 3 above)
Comment (parkr) [reply README.markdown]: All set up! You too, @DirtyF 😄 I went with first names – if you'd prefer something else, please let me know!

Comment (DirtyF) [reply README.markdown]: @parkr 🍷 cheers mate!

**Pull request merged**